src/hooks/docs.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Code, Icon, Text } from '@chakra-ui/react';
```

Comment:
This file should be called `useRtl.stories.tsx`. As new hooks will be added in the future in the `hooks` directory the name `docs.stories.tsx` would be too generic.